clang/lib/Sema/SemaDecl.cpp

```diff
@@ -11455,9 +11465,8 @@ static bool CheckTargetCausesMultiVersioning(Sema &S, FunctionDecl *OldFD,
   }
 
   // If this is 'default', permit the forward declaration.
-  if (!OldFD->isMultiVersion() &&
-      ((NewTA && NewTA->isDefaultVersion() && !OldTA) ||
-       (NewTVA && NewTVA->isDefaultVersion() && !OldTVA))) {
+  if ((NewTA && NewTA->isDefaultVersion() && !OldTA) ||
```

Comment:
Seems we are changing "target" attribute semantic here as well that we would like to avoid, why are we sure that !OldFD->isMultiVersion() is always true ?

Reply:
As far as I can tell yes, this is astatic function with exactly one call site.